truffle-config.js

```diff
@@ -24,54 +24,45 @@
 // const fs = require('fs');
 // const mnemonic = fs.readFileSync(".secret").toString().trim();
 
-module.exports = {
-  /**
-   * Networks define how you connect to your ethereum client and let you set the
-   * defaults web3 uses to send transactions. If you don't specify one truffle
-   * will spin up a development blockchain for you on port 9545 when you
-   * run `develop` or `test`. You can ask a truffle command to use a specific
-   * network from the command line, e.g
-   *
-   * $ truffle test --network <network-name>
-   */
+require('dotenv').config();
+const HDWalletProvider = require('truffle-hdwallet-provider-privkey');
+
+const INFURA_PROJECT_ID = process.env.INFURA_PROJECT_ID;
+const DEPLOYMENT_ACCOUNT_PK = process.env.DEPLOYMENT_ACCOUNT_PK;
```

Comment:
The `HDWalletProvider` expects the PK to not include the `0x` at the beginning. To avoid any errors, I think a good idea would be to remove the `0x` from the parameter if present. For example using: `process.env.DEPLOYMENT_ACCOUNT_PK.replace(/^0x/, "")`